### PR TITLE
[post] News in ConfigTransformer

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Hi @Tofandel,

I just wrote a post about new features in ConfigTransformer and included your great PRs too :) 

If you have a few mins and you want to review the content, what to add/remove, please be my guest :+1: I'd like to hear your input.

Thanks

**Comment by TomasVotruba** [comment]
Ready to ship... feel free to ad feedback post-merge, as change is just another commit :)

**Comment by Tofandel** [comment]
The output of an example is different of the input but otherwise it looks good 👍

**Comment by Tofandel** [comment]
Oh also, worth mentionning is the speed improvement of the const refactor, I just did a test before it was taking 3 seconds to convert all my config, now it takes 200ms I don't even have time to see the output of the command

**Comment by TomasVotruba** [comment]
> The output of an example is different of the input but otherwise it looks good +1

Could you comment exact line you mean? Apart spacing of course :) I keep human-readable ones, so it's easier to read.

> Oh also, worth mentionning is the speed improvement of the const refactor

Sounds huge! I'll add it :+1:

**Comment by Tofandel** [comment]
@TomasVotruba It's in the review of this PR

**Comment by TomasVotruba** [comment]
@Tofandel I don't see that, just your 3 comments here :) 

It might need to submit again, Github Reviews sometimes need double submit.
Could you share screenshot?

**Comment by Tofandel** [comment]
image

**Comment by TomasVotruba** [comment]
Yeah, that happened to me million times :) 

If it's "pending", only you can see it. You have to submit it. The button should be in the top somewhere.

**Comment by Tofandel** [comment]
Indeed, never knew that, github could add some tooltip one the `pending` to explain that

**Comment by TomasVotruba** [comment]
Yeah, this is the worst Github experience :laughing: 

I've made many review and then complained, why people didn't resolve or reply those. Are they blind or what? There is 10 comments! :laughing:

File: packages/blog/data/2022/2022-07-18-8-news-in-config-transformer-that-converts-symfony-yml-to-php.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\ConfigTransformer\Tests\Converter\ConfigFormatConverter\YamlToPhp\YamlToPhpTest;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $parameters = $containerConfigurator->parameters();
+
+    $parameters->set('class_constant', YamlToPhpTest::TEST);
+    $parameters->set('class', YamlToPhpTest::class);
+    $parameters->set('unexisting_constant', SomeClass::Constant);
+    $parameters->set('another_key', '%env(string:default::CODE_EDITOR)%');
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply packages/blog/data/2022/2022-07-18-8-news-in-config-transformer-that-converts-symfony-yml-to-php.md:209]: Thanks :+1: Just pushed the fix https://github.com/TomasVotruba/tomasvotruba.com/commit/3bf3ee49d38157133e0c7a472b5cc5445bdedc7c

**Pull request merged**